pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/RepoManager.java

```diff
@@ -349,17 +349,22 @@ private RepositoryData repackage(File sourceTopLevelDirectory) {
 
         ParentPomDownloader.addParentPoms(targetRepoContentsDir.toPath());
 
+        // TODO: testMode is need for ResolveOnlyRepositoryTest. These aren't mocked and cause the test to fail.
         if (!isTestMode) {
             RepositoryUtils.removeCommunityArtifacts(targetRepoContentsDir);
             RepositoryUtils.removeIrrelevantFiles(targetRepoContentsDir);
+        }
 
+        if (generationData.getStrategy().equals(RepoGenerationStrategy.RESOLVE_ONLY)) {
             // Delete excluded artifacts in the maven-repository. Needed for resolve only generation where filtering
```

Comment:
Added if block to match comment for code added in https://github.com/project-ncl/bacon/pull/909